master-nonlatent/master.cfg

```diff
@@ -117,7 +117,7 @@ dockerlibrary_worker = mkWorker(
     os.getenv("MASTER_NONLATENT_DOCKERLIBRARY_WORKER", default="bb-rhel8-docker"),
     properties={
         "jobs": 1,
-        "save_packages": os.getenv("ENVIRON") == "PROD",
+        "save_packages": os.getenv("ENVIRON") != "DEV",
```

Comment:
why do we need a different logic for `save_packages` on dev/prod? Packages should be saved on both independently. Is there anything in particular that happens for dockerlibrary?

Reply:
as discussed on Zulip, this is normal, we do not want something from the dev environment to be pushed on quay.io or at least it should use a specific tag.